test/system/710-kube.bats

```diff
@@ -157,4 +157,44 @@ status  | =  | {}
     run_podman rmi $(pause_image)
 }
 
+# kube play --wait=true, where we clear up the created containers, pods, and volumes when a kill or sigterm is triggered
+@test "podman kube play --wait" {
+    cname=c$(random_string 15)
+    fname="/tmp/play_kube_wait_$(random_string 6).yaml"
+    run_podman container create --name $cname $IMAGE top
+    run_podman kube generate -f $fname $cname
+
+    # delete the container we generated form
+    run_podman rm -f $cname
+
+    # force a timeout to happen so that the kube play command is killed
+    # and expect the timeout code 124 to happen so that we can clean up
+    local t0=$SECONDS
+    PODMAN_TIMEOUT=15 run_podman 124 kube play --wait $fname
+    local t1=$SECONDS
+    local delta_t=$((t1 - t0))
+    assert $delta_t -le 20 \
+           "podman kube play did not get killed within 10 seconds"
+
+    # there should be no containers running or created
+    run_podman ps -aq
+    is "$output" "" "There should be no containers"
+}
+
+# default kube play, where --wait=false and we don't clean up the created containers, pods, and volumes
+@test "podman kube play" {
```

Comment:
Does this add anything that isn't already tested in `700-play.bats`?

Reply:
ah nope, I missed that file. I also moved my test into 700-play.bats